### DISCUSSION TITLE
关于插件系统

### DISCUSSION THREAD

**Discussion by mrsai** [Milkdown/milkdown#1010] [General]
### Problem

新版本之后后，插件都被独立出来了，这个思路非常好，但是依据目前的文档来看，自己编写一款插件的成本非常高。
我使用vue来进行开发，我把slash插件从react转成了vue，但是花了比较长的时间。
我试图将table也转化为vue插件的时候，有点崩溃。

### Solution

所以，如果文档不能详细，能否丰富一些example中的示例，如image，table，task等。。。

**Comment by s524797336** [comment]
这个可能真得靠社区了，作者未必有多少时间，基于 Milkdown 的开源项目越多，这样的例子就越多。

**Reply by mrsai** [reply]
是啊，觉得做开源的人真的都超级nice，牺牲自己的时间来帮助别人。
但是现在有一个死循环，上手难度高，用的人就少，用的人少就社区不活跃。社区不活跃，用的人就越少。
我看作者在旧版本中其实这些功能都是集成的，后来拆出来让大家更自由的发挥，但是有能力发挥的人不多啊。
强烈建议编辑器和插件不要与框架强绑定。。。
